Sporadic test failure in metric_test.go

Sometimes running the test suite yields a failure. Running the tests with the same ginkgo seed doesn't reproduce it for me, so we have to make an educated guess.

```
Running Suite: gobrake
======================
Random Seed: 1579241484
Will run 31 of 31 specs

••••••
------------------------------
• Failure [0.206 seconds]
metric with real clock
/Users/kyrylo/code/airbrake/gobrake/metric_test.go:19
  supports measuring spans in goroutines [It]
  /Users/kyrylo/code/airbrake/gobrake/metric_test.go:20

  Expected
      <time.Duration>: 210356472
  to be within 10ms of ==
      <time.Duration>: 200000000

  /Users/kyrylo/code/airbrake/gobrake/metric_test.go:46
------------------------------
••••••••••••••••••••••••

Summarizing 1 Failure:

[Fail] metric with real clock [It] supports measuring spans in goroutines
/Users/kyrylo/code/airbrake/gobrake/metric_test.go:46

Ran 31 of 31 Specs in 1.364 seconds
FAIL! -- 30 Passed | 1 Failed | 0 Pending | 0 Skipped
--- FAIL: TestGobrake (1.36s)
FAIL
exit status 1
FAIL    github.com/airbrake/gobrake/v4  1.621s
```

--- a/metric_test.go
+++ b/metric_test.go
@@ -42,9 +42,9 @@ var _ = Describe("metric with real clock", func() {
 		sp0.Finish()
 		metric.finish()
 
-		Expect(metric.groups["sp0"]).To(BeNumerically("==", 100*time.Millisecond, 10*time.Millisecond))
-		Expect(metric.groups["sp1"]).To(BeNumerically("==", 200*time.Millisecond, 10*time.Millisecond))
-		Expect(metric.duration()).To(BeNumerically("==", 200*time.Millisecond, 10*time.Millisecond))
+		Expect(metric.groups["sp0"]).To(BeNumerically("==", 100*time.Millisecond, 20*time.Millisecond))
+		Expect(metric.groups["sp1"]).To(BeNumerically("==", 200*time.Millisecond, 20*time.Millisecond))
+		Expect(metric.duration()).To(BeNumerically("==", 200*time.Millisecond, 20*time.Millisecond))
 	})
 })
 

